dell/xps/17-9700/common.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, pkgs, ... }:
+
+{
+  # Boot loader
+  boot.kernelParams = lib.mkDefault [ "acpi_rev_override" ];
+  boot.kernelModules = lib.mkDefault [ "kvm-intel" ];
+
+  hardware.enableRedistributableFirmware = lib.mkDefault true;
+
+  # This will save you money and possibly your life!
+  services.thermald.enable = lib.mkDefault true;
+
+  boot.kernelPatches = [{
+    name = "enable-soundwire-drivers";
+    patch = null;
+    extraConfig = ''
+      SND_SOC_INTEL_USER_FRIENDLY_LONG_NAMES y
+      SND_SOC_INTEL_SOUNDWIRE_SOF_MACH m
+      SND_SOC_RT1308 m
+    '';
+    ignoreConfigErrors = true;
```

Comment:
What error are you seeing here?

Reply:
Good catch, seems to pass fine without now:
4a0cdc97c85cb941e8ff4a91226bbc92ca20cd9e